lib/install.js

```diff
@@ -133,8 +139,8 @@ async function installAll () {
     ['linux', '32'],
     ['linux', '64'],
     ['win', '32'],
-    ['mac', '32']
   ];
+  plats.push(parseFloat(CD_VER) < 2.23 ? ['mac', '32'] : ['mac', '64']);
```

Comment:
I feel like this line deserves some kind of comment, like "Chromedriver after 2.23 comes in a 64-bit version as well", or whatever is appropriate